### ISSUE TITLE
Refreshing (f5) is not performed when selecting a dropdown

### ISSUE THREAD

**Issue by guilleggomez** [reactstrap/reactstrap#1453]
- components: `Dropdown`
- reactstrap version `^7.1.0`
- react version `^16.2.0`
- bootstrap version `^4.1.3`

### What is happening?

When selecting an option from a dropdown (to expand or collapse that dropdown), input still is over component and refresh fro keyboard (pressing f5) is not available, any action is performed.

### What should be happening?

When pressing on the Dropdown component, it should allow the user to refresh the page when pressing on f5 from the keyboard (it is important for a page with sensible data)

### Steps to reproduce issue

1. Navigate to Dropdown component
2. Press on component (to expand or collapse)
3. Press on f5 to refresh page

### Code

<!--
https://reactstrap.github.io/components/navbar/   - Pressing on Dropdown and then refreshing
